lib/src/injector_base.dart

```diff
@@ -108,51 +59,56 @@ class Injector {
   /// A circular dependency is detected when the factory id was not removed
   /// meaning that the instance was not created
   /// but the same factory was called more than once
-  var _factoryCallIds = List<int>();
+  final _factoryCallIds = <int>[];
 
   /// Returns the registered dependencies with the signature of [T] and
-  /// [dependencyName].
+  /// the optional [dependencyName].
   ///
   /// Throws [NotDefinedException] when the requested dependency has not been
   /// registered yet.
   ///
   /// Throws [CircularDependencyException] when the injector detected a circular
   /// dependency setup.
-  T getDependency<T>({String dependencyName = ""}) {
+  T get<T>({String dependencyName = ""}) {
     _checkValidation<T>();
 
-    String identity = _getIdentity<T>(dependencyName);
+    final identity = _getIdentity<T>(dependencyName);
 
     if (!_factoryMap.containsKey(identity)) {
       throw NotDefinedException(type: T.toString());
     }
 
-    var factory = _factoryMap[identity];
-    var factoryId = factory.hashCode;
+    final factory = _factoryMap[identity];
+    final factoryId = factory.hashCode;
 
     if (_factoryCallIds.contains(factoryId)) {
       throw CircularDependencyException(type: T.toString());
     }
 
     _factoryCallIds.add(factoryId);
 
-    var instance = factory.instance as T;
+    final instance = factory.instance as T;
     _factoryCallIds.remove(factoryId);
 
     return instance;
   }
 
+  /// Shorter syntax for [get].
+  T call<T>({String dependencyName = ""}) => this.get<T>(dependencyName: dependencyName);
```

Comment:
```get``` is shorter than ```call``` :0

Reply:
We did you added this method?

Reply:
Call is the operator so we can use the injector similar to get it :

```dart

final car = injector.get<Car>();

final car2 = injector<Car>(); // <--- this is call
```